src/Dapr.Actors/Runtime/ActorHost.cs

```diff
@@ -40,11 +42,12 @@ public sealed class ActorHost
         /// </summary>
         public ActorId Id { get; }
 
-        internal DaprStateProvider StateProvider { get; }
-
         /// <summary>
         /// Gets the LoggerFactory for actor service
         /// </summary>
         public ILoggerFactory LoggerFactory { get; }
+
+        internal DaprStateProvider StateProvider { get; }
+        internal JsonSerializerOptions JsonSerializerOptions { get; }
```

Comment:
This property can be public. Generally if you pass something into a public constructor, it's just good style to make accessible as a readonly property. You're not really abstracting anything by hiding it.